templates/comments/list.html

```diff
@@ -91,6 +91,7 @@ <h2><i style="padding-right: 0.3em" class="fa fa-comments"></i>{{ _('Comments')
                                                    class="edit-link"><i class="fa fa-pencil fa-fw"></i></a>
                                             {% endif %}
                                             <a data-featherlight="{{ url('comment_votes_ajax', node.id) }}"
+                                               href="javascript:void(0)"
```

Comment:
Two lines below, `javascript:`. We should probably choose one of them instead of having both for consistency.

Reply:
Ah, you're right. There's a variety of options: `javascript:`, `javascript:void`, `javascript:void(0)` and all of them can also have brackets on the end.

I don't know what's better.